Sming/Makefile-windows.mk

```diff
@@ -9,7 +9,7 @@ SDK_BASE	?= $(ESP_HOME)/ESP8266_SDK
 SDK_TOOLS	 ?= $(ESP_HOME)/utils
 
 # Other tools mappings
-ESPTOOL		 ?= $(SDK_TOOLS)/esptool.exe
+ESPTOOL		 ?= $(SDK_TOOLS)/ESP8266/esptool.exe
```

Comment:
Could not find anything in the registry. Also no env variable. We could use perl/awk to parse changelog for the first occurence of regex of version string but it's a little dirty...

Created [issue](https://github.com/CHERTS/esp8266-devkit/issues/38)